game/templates/game/board.html

```
@@ -260,6 +260,20 @@
                     <span class="placeholder">No moves yet</span>
                 </div>
             </div>
+            <div class="card">
+                <div class="card-title">Manual Move Entry</div>
+                <div style="display: flex; flex-direction: column; gap: 8px;">
+                    <input type="text" id="algebraicMoveInput" placeholder="Enter move (e.g., e4, Nf3, O-O)" 
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)